### ISSUE TITLE
Learning Center Dashboard

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#3024]
### User Story

As a content author, who contributes Learning Center content, I need a quick access to existing LC content as well as LC top tasks, so I can perform my daily tasks efficiently.

### Affected users and stakeholders

* CMS editors of Learning Center product

### Hypothesis

_A hypothesis may depend on a spike ticket to be completed._

We believe that _this_solution_ will achieve _this_outcome_. We'll know that to be true when _this measurable outcome occurs._

### Assumptions
* Tk
* Tk

(How will these assumptions be validated?)

## Acceptance Criteria
* Testable_Outcome_X
* Testable_Outcome_Y
* Testable_Outcome_Z

**Comment by EWashb** [comment]
@jilladams I found this is the depths of the backlog. Yall are still R&S right? I'm also ok with closing this old epic if you have something newer.

**Comment by jilladams** [comment]
Yes we are and no we do not, afaik. FYI @wesrowe , moving into our backlog.

**Comment by wesrowe** [comment]
This seems like an Icebox item. Adding label.

**Comment by FranECross** [comment]
Closing as a no op
